### DISCUSSION TITLE
In which file or blueprint can I find "AAlsCharacterExample::Input_OnSprint" and "AAlsCharacterExample::Input_OnWalk"?

### DISCUSSION THREAD

**Discussion by ItaloLemosGasparo** [Sixze/ALS-Refactored#288] [Q&A]
Idk where can i change the "InputGraph"
I want the character to walk by default instead of running, and just run while Shift is pressed.
And after he Sprint I want him to walk again instead of running too.

**Comment by MagentaPuppet** [comment]
In the AlsCharacterExample cpp file as the name implies. Find the B_Als_Character blueprint and click the parent class in the top right corner. You'll of course need to have visual studio or sth equivalent to edit the code. It should open up AlsCharacterExample.h. Scroll down towards the end of the file to the last private section, right click on the Input_OnSprint() and click "Go To Definition". In the Input_OnSprint function, change where it says Running to Walking and there you go. 
If you want it to walk by default, search for "desired gait" in the B_Als_Character blueprint class defaults and change the default value to walking.
